bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
+}
+
 heroku-compress-assets() {
   puts-step "Compressing assets"
-  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress | indent
+  PATH=$PATH:/app/$NODE_PATH/bin .heroku/python/bin/python manage.py compress --force | indent
```

Comment:
Because during testing of the deployment it was useful for me to deploy with `DEBUG=True`. In which case, `COMPRESS_ENABLED` defaults to `not DEBUG` (i.e. `false`), thereby causing the `compress` command to fail with the error:

```
CommandError: Compressor is disabled. Set the COMPRESS_ENABLED setting or use --force to override.
```